tyr/servers/mongo/arbiter.py

```diff
@@ -2,8 +2,8 @@
 
 class MongoArbiterNode(MongoReplicaSetMember):
 
-    NAME_TEMPLATE = '{envcl}-rs{replica_set}-{zone}-arb'
-    NAME_SEARCH_PREFIX = '{envcl}-rs{replica_set}-{zone}-'
+    NAME_TEMPLATE = '{envcl}-rs{replica_set}-{location}-arb'
```

Comment:
The location isn't equivalent to the placement. The placement would be `us-east-1c` while the location would be `use1c`.